### ISSUE TITLE
pypi page: link to changelog

### ISSUE THREAD

**Issue by cbm755** [pymupdf/PyMuPDF#1236]
I notice many projects have a link to the changelog directory on pypi.  See for example [WeasyPrint](https://pypi.org/project/weasyprint/):

![Screenshot from 2021-08-29 11-17-51](https://user-images.githubusercontent.com/818622/131261172-dcb15cdd-9ca4-4e3c-8279-9ef967379d34.png)

Maybe PyMuPDF should have that too?  

I guess the appropriate link is https://pymupdf.readthedocs.io/en/latest/changes.html

**Comment by JorjMcKie** [comment]
Good idea - will check that out.
Thank you.

**Comment by JorjMcKie** [comment]
new version 1.18.18 now on PyPI

**Comment by JorjMcKie** [comment]
Hi @cbm755 - I must admit, that the result is presumably not what you would have liked to see.

I have looked up what is needed to achieve nicer results: apparently no less than a complete change of the setup config.
A new file `setup.cfg` must be created which replaces file PKG-INFO and whatever more detail info inside the `setup.py`. That new file is obviously the one responsible for those nicer entries you showed in your pasted image.
I keep working on that ...

**Comment by cbm755** [comment]
I'm personally happy!  I went to pypi, ctrl-f "changelog", click, done :-)

Only thing I noticed was first time I did that I got the older changelog (I assume b/c I caught it before you updated the readthedocs site).

FWIW, I believe you're right about moving away from `setup.py`.

**Comment by JorjMcKie** [comment]
> I'm personally happy! I went to pypi, ctrl-f "changelog", click, done :-)

Uff - good! Thanks!

I have been digging a bit more into this and studied the `setuotools` documentation for examples for how this should be done. Did not find any. But I did find find a package example: `rst2pdf`, and - *heureka* - saw that there is a parameter `project_urls`, which accepts a dictionary of keys like "Changelog" with URLs as values.
Once I had that, I also found it in the setuptools docu via search, saying 
![grafik](https://user-images.githubusercontent.com/8290722/133935373-9471d034-e8a8-4534-bec6-987cadec9ac0.png)
So I have to say again: "in the next version you will see ..."
